official/transformer/transformer_estimator_benchmark.py

```diff
@@ -194,21 +201,47 @@ def benchmark_graph_8_gpu(self):
     FLAGS['bleu_source'].value = self.bleu_source
     FLAGS['bleu_ref'].value = self.bleu_ref
     FLAGS.param_set = 'base'
-    FLAGS.batch_size = 2048 * 8
+    FLAGS.batch_size = 3072 * 8
```

Comment:
Why 3072?

Reply:
2048 is too small the goal is over 20K tokens per batch.  It might work if I train longer, but I would rather try 3072 (4096 * 8 was close ~27.226  around 75K steps) and still do 100K steps.  If that works I can try other configs and do more research or move look at the other hparams and maybe try 4096 again.